### PR TITLE
Added documentation for create work package endpoints

### PR DESCRIPTION
**Pull request by Azure7111**
This PR will introduce functionality to create work packages via APIv3.

https://community.openproject.org/work_packages/19987

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://azure7111-feature-create-work-package.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://azure7111-feature-create-work-package.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: doc/apiv3-documentation.apib [hunk 1]
```diff
@@ -1223,6 +1223,13 @@ The request body is the actual string that shall be rendered as HTML string.
 
 # Group Projects
 
+## Actions:
+
+| Link                       | Description                                                          | Condition                         |
+|:--------------------------:|----------------------------------------------------------------------| --------------------------------- |
+| createWorkPackage          | Form endpoint that aids in preparing and creating a work package     | **Permission**: add work packages |
+| createWorkPackageImmediate | Directly creates a work package in the project                       | **Permission**: add work packages |
+
```
Comment (NobodysNightmare) [inline doc/apiv3-documentation.apib:1232]: You should include that in the example for better clarity ;-)

File: doc/apiv3-documentation.apib [hunk 2]
```diff
@@ -1349,6 +1382,106 @@ but are also limited to the projects that the current user is allowed to see.
...
+                        "attribute": "Subject"
+                    }
+                }
+            }
+
+## Work Package Create Form [/api/v3/projects/{id}/work_packages/form]
+
+This endpoint returns a form to allow a guided creation of a new work package.
+
+For more details and all possible responses see the general specification of [Forms](#forms).
```
Comment (NobodysNightmare) [inline doc/apiv3-documentation.apib:1475]: You could point out that the payload will be prefilled with the default values for fields where default values are available.

**Comment by NobodysNightmare** [comment]
@0xF013 @floriank @myabc Any further thoughts on that?

The basic idea is to keep the flow similar/equivalent to the updating of a work package. Notable things coming to my mind:
- the payload will be prefilled with default values where applicable
- endpoints are rooted at `/api/v3/projects/:id/work_packages/` because they allow adding a work package to the given project (no free project choice inside the form/post)

**Comment by NobodysNightmare** [comment]
If there are no further thoughts I will merge this at the end of the day.

**Pull request merged**